src/commands/update/ip-blocks/put_ipblock_tag.go

```diff
@@ -3,10 +3,13 @@ package ip_blocks
 import (
 	"github.com/spf13/cobra"
 	"phoenixnap.com/pnapctl/common/client/ip"
+	"phoenixnap.com/pnapctl/common/models/ipmodels"
 	"phoenixnap.com/pnapctl/common/printer"
 	"phoenixnap.com/pnapctl/common/utils"
 )
 
+var Filename string
+
 const commandName = "put ip-block tag"
 
 var PutIpBlockTagCmd = &cobra.Command{
```

Comment:
This command is missing flags and an example.

Reply:
Added an example with flags

Reply:
Where are the flags being set?